xbmc/guilib/GUIFontManager.cpp

```diff
@@ -327,17 +332,19 @@ CGUIFont* GUIFontManager::GetDefaultFont(bool border)
   if (font13index == m_vecFonts.size())
   {
     if (m_vecFonts.empty())
-      return NULL;
+      return nullptr;
     font13index = 0;
   }
 
   if (border)
   {
     if (!font13border)
     { // create it
-      CGUIFont *font13 = m_vecFonts[font13index];
+      CGUIFont* font13 = m_vecFonts[font13index];
       OrigFontInfo fontInfo = m_vecFontInfo[font13index];
-      font13border = LoadTTF("__defaultborder__", fontInfo.fileName, 0xFF000000, 0, fontInfo.size, font13->GetStyle(), true, 1.0f, fontInfo.aspect, &fontInfo.sourceRes, fontInfo.preserveAspect);
+      font13border = LoadTTF("__defaultborder__", fontInfo.fileName, 0xFF000000, 0, fontInfo.size,
+                             font13->GetStyle(), true, 1.0f, fontInfo.aspect, &fontInfo.sourceRes,
+                             fontInfo.preserveAspect);
```

Comment:
`0xFF000000` can be replaced by `UTILS::COLOR::BLACK`

Reply:
Thanks for the note, has changed to `UTILS::COLOR::BLACK`.